test/Renci.SshNet.IntegrationTests/KeyExchangeAlgorithmTests.cs

```diff
@@ -22,6 +22,22 @@ public void TearDown()
             _remoteSshdConfig?.Reset();
         }
 
+        [TestMethod]
+        [Ignore]
```

Comment:
The OpenSSH at Ubuntu is not the latest version.
My local test passed with OpenSSH 9.9p1.